dali/python/nvidia/dali/ops.py

```diff
@@ -314,14 +317,19 @@ def __init__(self, inputs, op, **kwargs):
         # Argument inputs
         for k in sorted(kwargs.keys()):
             if k not in ["name"]:
-                if not isinstance(kwargs[k], _EdgeReference):
+                arg_inp = kwargs[k]
+                if arg_inp is None:
+                    continue
+                if isinstance(type(arg_inp), _ScalarConstant):
+                    arg_inp = instantiate_constant_node(arg_inp)
```

Comment:
I can only see `_instantiate_constant_node` (starting with underscore). Is this codepath tested?

Reply:
Bug. Fixed.